### ISSUE TITLE
compile error on Debian GNU/Linux and R 4.0.4

### ISSUE THREAD

**Issue by egonw** [sysrev/rsr#23]
I get this syntax error:

```
Installing package into ‘/home/user/R/x86_64-pc-linux-gnu-library/4.0’
(as ‘lib’ is unspecified)
* installing *source* package ‘rsr’ ...
** using staged installation
** R
Error in parse(outFile) : 
  /tmp/RtmpANcfx0/R.INSTALL1f3b141546b47/rsr/R/api.R:20:38: unexpected input
19:     a = unlist(res$error.srp) 
20:     a = set_names(a,map_chr(names(a),\
                                         ^
ERROR: unable to collate and parse R files for package ‘rsr’
* removing ‘/home/user/R/x86_64-pc-linux-gnu-library/4.0/rsr’
Warning message:
In i.p(...) :
  installation of package ‘/tmp/Rtmp4A2INt/file1ef61747615ee/rsr_0.4.0.9002.tar.gz’ had non-zero exit status
```

R info:

```
R version 4.0.4 (2021-02-15) -- "Lost Library Book"
Copyright (C) 2021 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)
```

**Comment by egonw** [comment]
I think the `DESCRIPTION` needs updating to depend on R 4.1 or even 4.2?

https://github.com/sysrev/rsr/blob/master/DESCRIPTION#L6

**Comment by egonw** [comment]
I can confirm I can install the package with R 4.2.2, but the minimal R version can still be 4.1.

**Comment by tomlue** [comment]
Thank you egon, I will make this update soon

**Comment by tomlue** [comment]
closed by 8bbc8a0 minimal version of R set to 4.1
